Miscellaneous operations doc fixes (#5598)
microsoft/CCF: doc/operations/ledger_snapshot.rst

Once a snapshot has been generated by the primary, operators can copy or mount the `read-only` snapshot directory to the new node directory before it is started. On start-up, the new node will automatically resume from the latest committed snapshot file in the ``snapshots.directory`` directory. If no snapshot file is found, all historical transactions will be replicated to that node.

> It is important to note that new nodes cannot resume from a snapshot and join a service via a node that started from a more recent snapshot. For example, if a new node resumes from a snapshot generated at ``seqno 50`` and joins from a (primary) node that originally resumed from a snapshot at ``seqno 100``, the new node will throw a ``StartupSeqnoIsOld`` error shortly after starting up. It is expected that operators copy the *latest* committed snapshot file to new nodes before start up.

Author: I believe this was the wrong way around all along.

Reviewer: ```suggestion
It is important to note that new nodes cannot join a service if the snapshot they start from is more recent than the snapshot of the node they join via. For example, if a new node resumes from a snapshot generated at ``seqno 50`` and joins from a (primary) node that originally resumed from a snapshot at ``seqno 100``, the new node will throw a ``StartupSeqnoIsOld`` error shortly after starting up. It is expected that operators copy the *latest* committed snapshot file to new nodes before start up.
```

Author: I think "than the snapshot of the node" is ambiguous (it may mean the latest snapshot generated by that node?). 

What about "It is important to note that new nodes cannot join a service if the snapshot they start from is older than the snapshot the primary node started from."?

Reviewer: Oh, is it the primary? Or the joinee?

Reviewer: Sure, let's say primary rather than "node they join via" then, and your wording works.
